tests/acceptance/features/webUIFiles/createFolders.feature

```diff
@@ -0,0 +1,38 @@
+@webUI @insulated @disablePreviews
```

Comment:
lets get rid of  `@disablePreviews` for now, that tag is not used in the phoenix tests yet